docs/content.zh/docs/connectors/datastream/formats/parquet.md

```diff
@@ -198,14 +198,14 @@ ds = env.from_source(source, WatermarkStrategy.no_watermarks(), "file-source")
 
 Flink 支持三种方式来读取 Parquet 文件并创建 Avro records （PyFlink 只支持 Generic record）：
 
-- [Generic record](https://avro.apache.org/docs/1.10.0/api/java/index.html)
-- [Specific record](https://avro.apache.org/docs/1.10.0/api/java/index.html)
-- [Reflect record](https://avro.apache.org/docs/1.10.0/api/java/org/apache/avro/reflect/package-summary.html)
+- [Generic record](https://avro.apache.org/docs/++version++/api/java/org/apache/avro/generic/package-summary.html)
```

Comment:
This PR refers to the latest version of Avro instead of each minor version in all places (see below). Note the version-specific link is not working currently: `https://avro.apache.org/docs/1.11.4/api/java/index.html` Also it gets stale if we put version specific links when we upgrade Avro version in `pom.xml`.

Reply:
Thanks David!

As most places in the Flink doc, in this PR we link all places to the latest version instead of minor versions. That assumes we do not have big lag in adopting the Apache Avro versions, and users can always find version-specific doc in Avro doc page.